__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const app = require('../app');
+const request = require('supertest');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const data = require('../db/data/test-data');
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/topics", () => {
+    describe("GET", () => {
+        test("Status 200 - Returns array of topic objects", () => {
+            const objectType = "object";
+            const objectKeys = ['slug', 'description'];
+
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body)).toBe(false);
```

Comment:
I would say you should always assert what you want, not what you don't want. Else you'll be there all day!